packages/nouvelle/docs/index.md

```diff
@@ -7,7 +7,9 @@ position: 1
 
 # Nouvelle Theme Overview
 
-> Kendo UI Nouvelle Theme is currently in development stage and it is, currently, covering the base components. Please vote for it if you want a full coverage: https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables.
+> Kendo UI Nouvelle Theme is currently experimental and covers the base components such as buttons, inputs, etc. We have an official feature request that you can vote for if you want a full coverage: https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables. Once scoped and planned, we will continue with the implementation. The updates will be posted in [this public issue](https://github.com/telerik/kendo-themes/issues/798).
```

Comment:
The Kendo UI Nouvelle Theme is currently experimental and covers base components such as buttons, inputs, etc. You can vote for it in this [Feedback Portal feature request](https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables) if you want full coverage. Any updates will be posted in [this public issue](https://github.com/telerik/kendo-themes/issues/798).

Reply:
This sounds like the general way things go, but doesn't give any information, so I cut it: "Once scoped and planned, we will continue with the implementation."
If we want to give information about the stage of execution we are at, perhaps we can say "The full implementation of this theme is yet to be scoped and planned."